app/scripts/modules/amazon/src/instance/details/instanceDetails.html

```diff
@@ -235,6 +237,11 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
           >
           </copy-to-clipboard>
         </dd>
+        <dd ng-if="instance.ipv6Addresses.length" ng-repeat="ip in instance.ipv6Addresses">
+          <a href="http://{{ip}}:{{state.instancePort}}" target="_blank">{{ip}}</a>
+          <copy-to-clipboard class="copy-to-clipboard copy-to-clipboard-sm" text="ip" tool-tip="'Copy to clipboard'">
```

Comment:
Is this doing anything?  I don't see how the value of the IP is being passed to `copy-to-clipboard`

Reply:
It is passed using `text="ip"`, where `ip` comes from the `ng-repeat`.